### PR TITLE
Async get consumer

### PR DESCRIPTION
**Pull request by julianrubisch**
ensure `connectedCallback` waits for a consumer

### CODE REVIEW

File: javascript/action_cable.js [hunk 1]
```diff
@@ -1,7 +1,25 @@
 export let consumer
 
+const wait = () => new Promise(resolve => setTimeout(resolve, 1))
```
Comment (leastbad) [inline javascript/action_cable.js]: This isn't a change request, just a question... is this a scenario in which there's a meaningful distinction between `1` and `0`? If not, in theory we could get rid of the 2nd argument.

Comment (julianrubisch) [reply javascript/action_cable.js]: didn't try, gimme a sec

Comment (julianrubisch) [reply javascript/action_cable.js]: done!

**Comment by leastbad** [comment]
Nice catch!

**Pull request merged**